rmo-konflux-fix-and-pko-migration.md

```
@@ -0,0 +1,209 @@
+# RMO Konflux Migration Fix + PKO Migration
+
+## What broke
+
+MR !179357 changed `REGISTRY_IMG` in the saas file from the OLM catalog image to the operator binary image:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)